native/connector/virtmic.sh

```diff
@@ -62,10 +75,12 @@ tail -f "$fifoPath" | {
 
     # Read the new target node
     while read -r targetNodeSerial; do
+        UtilLog "[virtmic.sh] [Got FIFO Data] $targetNodeSerial"
         killMonitor
         disconnectInputs "$virtmicId"
-        setsid bash -- connect-and-monitor.sh "$virtmicPortFlId" "$virtmicPortFrId" "$targetNodeSerial" &
+        setsid bash -- connect-and-monitor.sh "$virtmicPortFlId" "$virtmicPortFrId" "$targetNodeSerial" 2>&1 >`UtilGetLogPathForFile 'connect-and-monitor.sh'` &
```

Comment:
Is the use of `>` instead of `>>` intentional?
I think we should use `>>` to preserve the content of multiple runs:

```suggestion
        setsid bash -- connect-and-monitor.sh "$virtmicPortFlId" "$virtmicPortFrId" "$targetNodeSerial" 2>&1 >>`UtilGetLogPathForFile 'connect-and-monitor.sh'` &
```